terraform/gcp/main.tf

```diff
@@ -30,7 +30,7 @@ resource "google_compute_firewall" "node-exporter-{{ name }}" {
 
   allow {
     protocol = "tcp"
-    ports    = ["9100"]
+    ports    = ["9100","9616"]
```

Comment:
Right, 9616 is an access to the 9615 exposed by the node, to collect the node metrics ( before was not possible to gather that data ). 
Both these ports are behind base_authentication, so them are not public at all: 
https://github.com/w3f/polkadot-secure-validator/blob/features-and-patches/ansible/roles/node-exporter/files/polkadot.nginx.conf#L6-L7
https://github.com/w3f/polkadot-secure-validator/blob/features-and-patches/ansible/roles/node-exporter/files/nginx.conf#L6-L7

What do you think ?

Reply:
So both are hidden behind the following credentials (inventory):

```yaml
node_exporter_user='node_exporter_user'
node_exporter_password='node_exporter_password'
```

Maybe document this somewhere? Or even rename those fields? From `node_exporter_*` to `prometheus_*`, maybe.

Reply:
I called them "nginx_user" and "nginx_pass" since they are governing the Nginix authentication mechanism... also added a dedicated role for that and extracted it from the node exporter role
https://github.com/w3f/polkadot-secure-validator/blob/cca7297c7f67c1092077cd9a66669c0780ff65dc/ansible/inventory.sample#L61-L63